bugbug/bug_features.py

```diff
@@ -134,6 +134,10 @@ def cleanup_fileref(text):
     return re.sub(r'\w+\.py\b|\w+\.json\b|\w+\.js\b|\w+\.jsm\b|\w+\.html\b|\w+\.css\b|\w+\.c\b|\w+\.cpp\b|\w+\.h\b', '__FILE_REFERENCE__', text)
 
 
+def cleanup_responses(text):
+    return re.sub('>.*?\\n', ' ', text)
```

Comment:
Wouldn't `>.*` or `>[^\n]+` be better? After matching `>`, you'd match all characters until a line terminator.